src/components/atoms/ModalLink/index.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import Link from "next/link";
+import { useSearchParams } from "next/navigation";
+import { ReactNode } from "react";
+
+interface Props {
+  href: string;
+  children: ReactNode;
+  className?: string;
+}
+
+function ModalLink({ href, children, className }: Props) {
+  const searchParam = useSearchParams();
+  const queryString = searchParam.toString() && `?${searchParam.toString()}`;
```

Comment:
타입 에러때문에 이렇게 처리하신 건지 잘 모르겠지만 이렇게 하시면 타입 에러도 안 날 것 같습니다!
```suggestion
  const queryString = `?${searchParam.toString() || ''}`;
```

Reply:
ModalLink 컴포넌트는 더이상 사용하지 않는 컴포넌트라 삭제 처리 했습니다